### PR TITLE
AA-427: Remove all mentions of 'aggregation' possible

### PR DESCRIPTION
**Pull request by forshtat**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/erc-4337.md [hunk 1]
```diff
@@ -222,11 +206,9 @@ this bundler is supposed to track the `key` and `sequence` pair of the UserOpera
 
 ### Required entry point contract functionality
 
-There are 2 separate entry point methods: `handleOps` and `handleAggregatedOps`
+There is one main entry point method: `handleOps`.
```
Comment (drortirosh) [inline ERCS/erc-4337.md]:
```suggestion
The entry point method is `handleOps`, which handles an array of userOps
```

File: ERCS/erc-4337.md [hunk 2]
```diff
@@ -222,11 +206,9 @@ this bundler is supposed to track the `key` and `sequence` pair of the UserOpera
...
 
-* `handleOps` handles userOps of accounts that don't require any signature aggregator.
-* `handleAggregatedOps` can handle a batch that contains userOps of multiple aggregators (and also requests without any aggregator)
-* `handleAggregatedOps` performs the same logic below as `handleOps`, but it must transfer the correct aggregator to each userOp, and also must call `validateSignatures` on each aggregator before doing all the per-account validation.
+* `handleOps` handles userOps of accounts.
```
Comment (drortirosh) [inline ERCS/erc-4337.md]:
```suggestion

```

File: ERCS/erc-4337.md [hunk 3]
```diff
@@ -357,7 +312,7 @@ To add a UserOperation into the mempool (and later to add it into a bundle) we n
 In addition, we need to verify that the same will hold true when executed on-chain.
 For this purpose, a UserOperation is not allowed to access any information that might change between simulation and execution, such as current block time, number, hash etc.
 In addition, a UserOperation is only allowed to access data related to this sender address: Multiple UserOperations should not access the same storage, so it is impossible to invalidate a large number of UserOperations with a single state change.
-There are 3 special contracts that interact with the account: the factory (initCode) that deploys the contract, the paymaster that can pay for the gas, and a signature aggregator (described later)
+There are 2 special contracts that interact with the account: the factory (initCode) that deploys the contract, and the paymaster that can pay for the gas.
```
Comment (drortirosh) [inline ERCS/erc-4337.md]:
```suggestion
There are 2 special entity contracts that interact with the account: the factory (initCode) that deploys the contract, and the paymaster that can pay for the gas.
```

File: ERCS/erc-4337.md [hunk 4]
```diff
@@ -583,13 +531,11 @@ The result `SHOULD` be set to the **userOpHash** if and only if the request pass
   * **code: -32503** - UserOperation out of time-range: either wallet or paymaster returned a time-range, and it has already expired (or will expire soon)
     * The `data` field SHOULD contain the `validUntil` and `validAfter` values
     * The `data` field SHOULD contain a `paymaster` value, if this error was triggered by the paymaster
-  * **code: -32504** - transaction rejected because paymaster (or signature aggregator) is throttled/banned
-    * The `data` field SHOULD contain a `paymaster` or `aggregator` value, depending on the failed entity
-  * **code: -32505** - transaction rejected because paymaster (or signature aggregator) stake or unstake-delay is too low
-    * The `data` field SHOULD contain a `paymaster` or `aggregator` value, depending on the failed entity
+  * **code: -32504** - transaction rejected because paymaster is throttled/banned
```
Comment (drortirosh) [inline ERCS/erc-4337.md]: irrelevant, as it is removed to an external ERC

Comment (forshtat) [reply ERCS/erc-4337.md]: Removed all mentions of error codes.

File: ERCS/erc-4337.md [hunk 5]
```diff
@@ -390,19 +344,15 @@ struct ReturnInfo {
...
-
 struct StakeInfo {
   uint256 stake;
   uint256 unstakeDelaySec;
 }
 
-
 ```
 
+The `AggregatorStakeInfo` structure is further defined in [ERC-XXXX](link).
```
Comment (shahafn) [inline ERCS/erc-4337.md:354]: TODO add actual link

**Comment by forshtat** [comment]
Closed in favour of https://github.com/ethereum/ERCs/pull/627

**Pull request closed without merging**